scripts/build-env.sh

```diff
@@ -7,100 +11,70 @@ else
     file="config.js"
 fi
 
-if [ -z ${GQL_URL+x} ]
-then
-    echo 'Error! $GQL_URL is required.'
-    exit 1
-fi
-
-if [ -z ${VERSION+x} ]
-then
-    echo 'Error! $VERSION is required.'
-    exit 1
-fi
-
-NETWORK="main"
-POLKADOT_URL="wss://tfchain.grid.tf/ws"
-PROXY_URL="https://gridproxy.grid.tf"
-ACTIVATION_SERVICE_URL="https://activation.grid.tf"
-EXPLORER_URL="https://explorer.grid.tf/"
-TFT_ASSET_ISSUER=GBOVQKJYHXRR3DX6NOX2RRYFRCUMSADGDESTDNBDS6CDVLGVESRTAC47
-BRIDGE_TFT_ADDRESS=GBNOTAYUMXVO5QDYWYO2SOCOYIJ3XFIP65GKOQN7H65ZZSO6BK4SLWSC
-STELLAR_HORIZON_URL="https://horizon.stellar.org"
-PLAYGROUND_URL="https://play.grid.tf"
-RELAY="relay.grid.tf"
+for i in "${REQUIRED_ENV_VARS[@]}"
+do
+    if ! [[ -v $i ]]; then
+        echo -e "\n\e[1;50m \e[1;31m$i is required!\e[0m\n \e[1;3mPlease set it by executing the following command."
+        echo -e "\e[1;31m export\e[0m \e[1;32m$i\e[0m=\e[1;38m'Your Value Here'\n"
+        return
+    fi
+done
 
 GRAVITY_CONTRACT_ADDRESS="0xBF8C35Ad93366E159C0F0B75F1a4f4ad6Ff80811"
 TFT_TOKEN_CONTRACT_ADDRESS="0xDC5a9199e2604A6BF4A99A583034506AE53F4B34"
 BRIDGE_FEES="3"
 TFT_DECIMALS=7
 TFT_DENOM="TFT"
-PROPOSAL_DENOM="TFT"
 COSMOS_REST="https://tfhub.test.grid.tf:1317/"
 TENDERMINT_RPC="https://tfhub.test.grid.tf:26657/"
 GAS_PRICE="80TFT"
 CHAIN_ID="threefold-hub-testnet"
 
-case $GQL_URL in
-  *"dev"*)
-    NETWORK="dev"
-    GQL_URL="https://graphql.dev.grid.tf/graphql"
-	  PROXY_URL="https://gridproxy.dev.grid.tf"
-    POLKADOT_URL="wss://tfchain.dev.grid.tf/ws"
+case $NETWORK in
+  "dev")
     TFT_ASSET_ISSUER=GA47YZA3PKFUZMPLQ3B5F2E3CJIB57TGGU7SPCQT2WAEYKN766PWIMB3
     BRIDGE_TFT_ADDRESS=GDHJP6TF3UXYXTNEZ2P36J5FH7W4BJJQ4AYYAXC66I2Q2AH5B6O6BCFG
-    ACTIVATION_SERVICE_URL="https://activation.dev.grid.tf"
     EXPLORER_URL="https://explorer.dev.grid.tf/"
     STELLAR_HORIZON_URL="https://horizon-testnet.stellar.org"
-    PLAYGROUND_URL="https://play.dev.grid.tf"
-    RELAY="relay.dev.grid.tf"
+    ;;
+  *"qa"*)
+    TFT_ASSET_ISSUER=GA47YZA3PKFUZMPLQ3B5F2E3CJIB57TGGU7SPCQT2WAEYKN766PWIMB3
+    BRIDGE_TFT_ADDRESS=GDHJP6TF3UXYXTNEZ2P36J5FH7W4BJJQ4AYYAXC66I2Q2AH5B6O6BCFG
```

Comment:
I think the bridge address should be configurable as well

Reply:
`BRIDGE_TFT_ADDRESS` and `TFT_ASSET_ISSUER`?

Reply:
`TFT_ASSET_ISSUER` depends on the stellar network, not on the tfchain